src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -488,7 +488,7 @@ public void AddClaims(IEnumerable<Claim> claims)
             }
         }
 
-        internal static object GetClaimValueUsingValueType(Claim claim)
+        private static object GetClaimValueUsingValueType(Claim claim)
```

Comment:
I'll revert these.
What's the reason they are marked as internals? The internals aren't used outside the class nor the assembly is decorated with `InternalsVisibleToAttribute`